x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -127,8 +97,25 @@ export function StickyTransactionProperties({
           defaultMessage: 'Errors'
         }
       ),
-      val: errorCount ? errorsOverviewLink : noErrorsText,
-      width: '25%'
+      val: errorCount ? (
+        <ErrorsOverviewLink
```

Comment:
Would it be possible to disable the tooltip showing "View 2 related errors" in this place, as I think the suffix label "Related errors" makes the tooltip superfluous?

<img width="502" alt="Screenshot 2019-03-14 at 09 05 54" src="https://user-images.githubusercontent.com/4104278/54341097-4965cf80-4639-11e9-8047-3ef03c6db60a.png">

Reply:
@ogupte Maybe the component should have a `verbose` options. If `true` it show the suffix label, if false it has the tooltip.

Reply:
> Ah, that looks great... when all the labels are aligned left. When the spans are further on the right, and the label information aligns on the right, the duration should show on the left, because it looks a little strange when we reserve the space for the duration to be shown. Not sure how easily achievable this is? I've tried to figure out how we align the labels, especially when the span is on on the right.

I think we should create a new ticket for this since there's some non-trivial to determine the alignment within the waterfall.

Reply:
@ogupte OK, let's move the show/hide functionality out of this PR too, I think it's too distracting as is, and not critical to implement the related errors count on the transaction span row. Perhaps we can open a PR with code, but have a task that has to look at the directional placement. What do you think?